src/EnergyPlus/GeneralRoutines.cc

```diff
@@ -442,10 +427,10 @@ void ControlCompOutput(EnergyPlusData &state,
         switch (SimCompNum) {      // Tuned If block changed to switch
         case ParallelPIUReheatNum: // 'AIRTERMINAL:SINGLEDUCT:PARALLELPIU:REHEAT'
```

Comment:
There are a lot of hacky functions in EnergyPlus. This one's in the top 10. Okay, maybe not. For sure top 50. This function-specific enum stuff is total nonsense.